superset-frontend/src/views/CRUD/data/database/DatabaseModal/index.tsx

```diff
@@ -1231,7 +1231,7 @@ const DatabaseModal: FunctionComponent<DatabaseModalProps> = ({
         onClick={() => {
           setLoading(true);
           fetchAndSetDB();
-          window.location.href = '/tablemodelview/list#create';
+          window.location.href = '/dataset/add/';
```

Comment:
The modal would auto-populate the database field with whatever database was just created – do we want to maintain that behavior?

Reply:
I think it should still do that, right? This only changes where the user is redirected when they click "CREATE DATASET" on the final step of the create database modal.